internal/core/data/v2/application/device_test.go

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package application
+
+import (
+	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
+	contract "github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+
+	"github.com/google/uuid"
+)
+
+var testEvent contract.Event
+
+const (
+	testDeviceName string = "Test Device"
+	testOrigin     int64  = 123456789
+	testUUIDString string = "ca93c8fa-9919-4ec5-85d3-f81b2b6a7bc1"
+)
+
+// Supporting methods
+// Reset() re-initializes dependencies for each test
+func reset() {
```

Comment:
> Expected to see tests related to device, but looks more like test helper functions for the package. Rename the file more appropriate?

Renamed to `test_utils.go`.

Reply:
we need xxx_test.go for the helper functions, and it won't be included to compiled binary.  These helper functions are for event only.  Why do we move to another file?

Reply:
@lindseysimple we can test checkDevice() by passing different `configuration.Writable.MetaDataCheck` to see whether it really call `mdc.CheckForDevice`

Reply:
@cloudxxx8 , good point on the proper naming of the file. My issue with the original name was that the contents it didn't have anything to do with testing `device`. Actual no unit tests at all.

Reply:
Just got @cloudxxx8 point about these for event tests. Should these simply put into the `event_test.go` file and remove this file rather than renaming again?

Reply:
Updated to put those util functions in `event_test.go` and removed `test_utils.go`. Thanks.